tests/test_federation.py

```diff
@@ -276,3 +276,111 @@ def test_cross_signing_keys_retry(self):
             "ed25519:" + remote_self_signing_key in self_signing_key["keys"].keys(),
         )
         self.assertTrue(remote_self_signing_key in self_signing_key["keys"].values())
+
+
+class StripUnsignedFromEventsTestCase(MessageAcceptTests):
+    def setUp(self):
+        super().setUp()
```

Comment:
This looks redundant---the superclass method will get called if this override is deleted.